src/reactive-dev-formatter.ts

```diff
@@ -67,18 +63,32 @@ const formatter: DevToolFormatter = {
     }
 };
 
+// Inspired from paulmillr/es6-shim
+// https://github.com/paulmillr/es6-shim/blob/master/es6-shim.js#L176-L185
+function getGlobal(): any {
```

Comment:
I'm fine with this! Maybe it is also time to start doing `globalThis` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) as part of this condition.